public/docs/ts/latest/tutorial/toh-pt1.jade

```diff
@@ -186,7 +187,7 @@ code-example(language="html").
 .l-main-section
 :marked
   ## The Road Ahead
-  Our Tour of Heroes only displays one hero and we really want to display a list of heroes.
+  Our Tour of Heroes only displays one hero, and we really want to display a list of heroes.
```

Comment:
This change is a matter of style, not grammar. More importantly, we have significant edits coming that will make this sentence irrelevant. Thanks for helping though.